deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1223,6 +1223,17 @@ fun(Conf) ->
     list_to_binary(cuttlefish:conf_get("queue_master_locator", Conf))
 end}.
 
+%% Queue leader locator (quorum queues and streams)
+%%
+
+{mapping, "queue_leader_locator", "rabbit.queue_leader_locator",
```

Comment:
I wish we could group this under `quorum_queues.` much like we have some new settings under `classic_queues.*` now but I see why you named it this way.